src/features/hotspots/setup/HotspotSetupBluetoothSuccess.tsx

```diff
@@ -52,19 +56,35 @@ const HotspotSetupBluetoothSuccess = () => {
     async (hotspot: Device) => {
       if (connectStatus === 'connecting') return
 
+      // Segment track for bluetooth connection
+      track(HotspotEvents.BLUETOOTH_CONNECTION_STARTED, {
+        hotspot_id: hotspot.id,
```

Comment:
Is there other information from the hotspot we can include, like the maker, model, or frequency?

Reply:
I am adding this feature.